Makefile

```diff
@@ -1079,3 +1085,18 @@ show_release_notes:
 .PHONY: spellcheck
 spellcheck:
 	bin/spell_check_java.sh
+
+$(JARS_JACOCO_AGENT):
+	mkdir -p $(@D)
+	curl $(JARS_JACOCO_AGENT_URL) --output $@
+	unzip $(JARS_JACOCO_AGENT) jacocoagent.jar -d $(BUILD_DIR)/jars
+
+$(JARS_JACOCO_CLI):
+	mkdir -p $(@D)
+	curl $(JARS_JACOCO_CLI_URL) --output $@
+
+coverage: $(FZ_INT) $(FZ_C) $(MOD_TERMINAL) $(MOD_JAVA_BASE) $(BUILD_DIR)/tests $(JARS_JACOCO_AGENT) $(JARS_JACOCO_CLI)
+	(FUZION_JAVA_OPTIONS="-javaagent:../../jars/jacocoagent.jar=destfile=../../jacoco.exec" $(FZ_SRC)/bin/run_tests.sh $(BUILD_DIR) int) || true
```

Comment:
The use of relative paths `../../jars/..` here is maybe a little brittle. Does `$(BUILD_DIR)/jars` work better?

Reply:
@fridis `$(BUILD_DIR)/jars` is a relative path as well, I think? Maybe we could do somethink like `$(realpath $(BUILD_DIR)/jars)`. But is this less brittle? I don't know.